src/main/java/com/google/ehub/data/EntertainmentItemList.java

```diff
@@ -0,0 +1,27 @@
+package com.google.ehub.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class EntertainmentItemList {
+  private final List<EntertainmentItem> itemList;
```

Comment:
nit: `itemList` can be renamed to `items`
We usually pluralize a variable name instead of adding List or Set.

Reply:
I renamed itemList variable and getItemList() to use the term 'items' in last commit.